Casks/b/beyond-compare@4.rb

```diff
@@ -0,0 +1,38 @@
+cask "beyond-compare@4" do
+  version "4.4.7.28397"
+  sha256 "a9ba4cea125bbfe00fa3e79de39937197ae5d479d94710f5b93da5bda377a0ce"
+
+  url "https://www.scootersoftware.com/BCompareOSX-#{version}.zip"
+  name "Beyond Compare"
+  desc "Compare files and folders"
+  homepage "https://www.scootersoftware.com/"
+
+  livecheck do
+    url "https://www.scootersoftware.com/download/v4"
+    regex(/BCompareOSX[_.-]v?(4\.{1}\d+(?:\.\d+)+)\.zip/i)
```

Comment:
```suggestion
    regex(/BCompareOSX[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

We should be able to stick to our usual patterns and keep this as it is, as we have already fixed the `livecheck` URL to v4.